src/components/Dashboard.js

```diff
@@ -4,487 +4,484 @@ class Dashboard extends Component {
-          <div className="container-fluid">
-
-              <div className="row">
-                  <div className="col-md-4">
-                      <div className="card card-chart">
-                          <div className="card-header" data-background-color="rose" data-header-animation="true">
-                              <div className="ct-chart" id="websiteViewsChart"></div>
-                          </div>
-                          <div className="card-content">
-                              <div className="card-actions">
-                                  <button type="button" className="btn btn-danger btn-simple fix-broken-card">
-                                      <i className="material-icons">build</i> Fix Header!
-                                  </button>
-                                  <button type="button" className="btn btn-info btn-simple" rel="tooltip" data-placement="bottom" title="Refresh">
-                                      <i className="material-icons">refresh</i>
-                                  </button>
-                                  <button type="button" className="btn btn-default btn-simple" rel="tooltip" data-placement="bottom" title="Change Date">
-                                      <i className="material-icons">edit</i>
-                                  </button>
-                              </div>
-                              <h4 className="card-title">Overall Progress</h4>
-                              <p className="category">Last Assessment</p>
-                          </div>
-                          <div className="card-footer">
-                              <div className="stats">
-                                  <i className="material-icons">access_time</i> assessment taken 2 days ago
-                              </div>
-                          </div>
-                      </div>
+      <div>
+        <div className="row">
+          <div className="col-md-4">
+            <div className="card card-chart">
+              <div className="card-header" data-background-color="rose" data-header-animation="true">
+                <div className="ct-chart" id="websiteViewsChart"></div>
+              </div>
+              <div className="card-content">
+                <div className="card-actions">
+                  <button type="button" className="btn btn-danger btn-simple fix-broken-card">
+                    <i className="material-icons">build</i> Fix Header!
+                    </button>
+                    <button type="button" className="btn btn-info btn-simple" rel="tooltip" data-placement="bottom" title="Refresh">
+                      <i className="material-icons">refresh</i>
+                    </button>
+                    <button type="button" className="btn btn-default btn-simple" rel="tooltip" data-placement="bottom" title="Change Date">
+                      <i className="material-icons">edit</i>
+                    </button>
```

Comment:
I notice that @tlambrou  remove the some className like `container-fluid`, so there's a big shift in the Dashboard file.

Reply:
I elevated these tags to wrap the components in the switch routes inside of the Sidebar component.  These tags are needed to prevent the NavTop and Footer components from collapsing into each other.